R/callOpenTiles.R

```diff
@@ -390,6 +397,14 @@ setMethod(
   # cellColDataCopy <- as.data.frame(dplyr::filter(data.frame(cellColData),
   # !!as.name(cellPopLabel) %in% cellPopulations))
   cellColDataCopy <- copy(cellColData)
+    
+  if(any(is.na(cellColDataCopy[[cellPopLabel]]))){
+  
+      warning(
+          stringr::str_interp("Some cells within the column ${cellPopLabel} are labeled as NA. Those cells will be ignored.")
+        )
+      
+  }
```

Comment:
This warning can also be moved earlier, and check `cellColData` (not the copy). 
can be moved to the top of `.callOpenTiles_default`.

Reply:
I moved it so that it's not on the copy.